ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/FieldType.java

```diff
@@ -17,5 +17,6 @@ public enum FieldType {
   MultiSelectList,
   Collection,
   Label,
-  CasePaymentHistoryViewer
+  CasePaymentHistoryViewer,
+  DynamicRadioList
```

Comment:
I thought you would need something like https://github.com/hmcts/ccd-config-generator/blob/master/ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/DynamicList.java 🤔

Reply:
@nitinprabhu I suggested this approach. We could do DynamicRadioList extends DynamicList {} - I'm not sure what is most intuitive in terms of API use.

Reply:
ohh makes sense DynamicRadioList extend DynamicList so we need to define a new class DynamicRadioList in config generator. Not sure if we need it in FieldType.java 🤔

Reply:
Sorry that's what I mean it could be done this way, by adding DynamicRadioList to FieldType and then using typeOverride on a `DynamicList<Something> field;` OR you could do it by just extending DynamicList with an empty implementation. I thought this way was a bit cleaner